ports/catwarrior/port.json

```
@@ -0,0 +1,29 @@
+{
+  "version": 3,
+  "name": "catwarrior.zip",
+  "items": [
+    "Cat Warrior.sh",
+    "catwarrior"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Cat Warrior",
+    "porter": [
+      "JanTrueno"
+    ],
+    "desc": "Cat Warrior is an action-platformer of retro and cute style about a cat knight on his way to recapture his stolen treasure. Deep into the dangerous fantasy world,defeat the challenging big boss with your slash, dash, jump, walljump and other skills to find the treasure back!",
+    "desc_md": null,
+    "inst": "Install portfiles. Buy game on Steam and download windows (and/or linux) version. Copy all files from the \"catwarrior\" folder to \"gamedata\" folder.",
+    "inst_md": null,
+    "genres": [
+      "action",
+      "platformer"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": "mono-6.12.0.122-aarch64.squashfs",
+    "reqs": [],
+    "arch": []
```

Not as important as missing the armhf tag but still :)
```suggestion
    "arch": [
        "aarch64"
    ]
```

---

No arch if you use a runtime. :D